Extensions/Video/videoruntimeobject.js

```diff
@@ -290,10 +322,9 @@ gdjs.VideoRuntimeObject.gdjsCallbackRuntimeSceneUnloaded = function(
 /**
  * When a scene is paused, pause any video being run.
  * @private
+ * @param {gdjs.RuntimeScene} runtimeScene The {@link gdjs.RuntimeScene} this objects has to be removed from.
```

Comment:
The {@link gdjs.RuntimeScene} that is being paused

Reply:
fixed